tests/test_collaborativerecommender.py

```diff
@@ -1,29 +1,68 @@
+"""
+Test cases for the TAAR core library.
+"""
+
 import pytest
 import responses
 
-from taar.recommenders.collaborative_recommender import CollaborativeRecommender, ADDON_MODEL_URL, ADDON_MAPPING_URL
+from taar.recommenders.collaborative_recommender import ADDON_MAPPING_URL
```

Comment:
I split the line just for PEP8 compliance.  While I generally tend to agree that we should not 'damage' the blame/history of the file because of formatting, I don't think we've accrued enough history for that to be a big deal yet.